components/enrichers/reachability/internal/fs/fs.go

```diff
@@ -0,0 +1,97 @@
+package fs
```

Comment:
perhaps this should also be part of the base enricher. not for this PR but maybe good to be in a ticket for the OSS refactoring project